### ISSUE TITLE
any reason this is hardcoded to usdc (as you can do both) (same for calls)

### ISSUE THREAD

**Issue by kjr217** [8ball030/rysk_explorations#55]
https://github.com/8ball030/rysk_explorations/blob/baac205c826d65035bec5a40d96f0d2e4025cba9/rysk_client/client.py#L491

**Comment by 8ball030** [comment]
currently, i have set it up to do covered, however I could probably knock a few of your issues on the head just by making collateral configurable :)

**Comment by kjr217** [comment]
yeah, you can easily determine the required collateral for partial collateralistion here: https://github.com/rysk-finance/dynamic-hedging/blob/rysk-beyond/packages/contracts/contracts/packages/opyn/new/NewCalculator.sol#L511 or here: https://github.com/rysk-finance/dynamic-hedging/blob/rysk-beyond/packages/contracts/contracts/packages/opyn/new/NewCalculator.sol#L367
